alliance/apps/backlog/forms.py

```diff
@@ -176,6 +176,19 @@ def save(self, request, commit=True, *args, **kwargs):
         backlog.priority = self.cleaned_data['priority']
         backlog.module = "backlog"
 
+        teamNameFromRequest = request.session.get('teamName')
+        logger.debug(teamNameFromRequest)
+
+        if 'Alliance Test Team' == teamNameFromRequest :
```

Comment:
Can we remove the first condition? That is, only express condition for 'North Stars' and 'All Megastars' and let everything else, including 'Alliance Test Team' fall through to the final condition